website/docs/terms/subquery.md

```diff
@@ -1,10 +1,14 @@
 ---
 id: subquery
 title: Subquery in SQL
+description: "A subquery is what the name suggests: a query within another query. The true inception of SQL. Subqueries are often used when you need to process data in several steps."
```

Comment:
is the `description` field supposed to be in quotes?

Reply:
If there is a semi-colon present, yes. Otherwise, it breaks the frontmatter format